src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRow1099Processor.java

```diff
@@ -1127,6 +1131,14 @@ private void processExclusionsAndAmounts(ResultSet rs, TransactionDetailRow deta
                         .append(paymentDateP.value).append(';')
                         .append(docNumberP.value).append(';')
                         .append(docLineNumberP.value).toString());
+        
+        //Determine if there is override of tax box due to the payment reason
+        if (getPaymentReason1099BoxService().isPaymentReasonMappedTo1099Box(paymentReasonCodeP.value)) {
+			String mappedBox = getPaymentReason1099BoxService().getPaymentReason1099Box(paymentReasonCodeP.value);
+			LOG.info("Overriding to tax box " + mappedBox + "  because of payment reason " + paymentReasonCodeP.value + " for vendor " + vendorNameForOutput);

```

Comment:
Should this be debug level, or is this something that we'll want to generally see in our logs?